src/utilities/isRequestResolvable.js

```diff
@@ -22,15 +23,18 @@ export default (context: string, requestMap: Object, request: string, parentFile
     return false;
   }
 
-  const absoluteTargetResourcePath = path.resolve(path.dirname(parentFilename), request);
+  const absoluteTargetResourcePath = resolvePath(path.dirname(parentFilename), request);
+  const contextAbsoluteResourcePath = resolvePath(path.dirname(context));
```

Comment:
we need matching absolute path resolution here as on Window, the absolute path of "/" will be to "C:\".